src/Documentation/Getting-Started-With-Orleans/Grain-Persistence.md

```diff
@@ -134,6 +134,9 @@ A quick way to test this is to (see in the aforementioned the few commented line
 The ADO.NET persistence has functionality to version data and define arbitrary (de)serializers with arbitrary application rules and streaming, but currently
 there is no method to expose them to application code. More information in [ADO.NET Persistence Rationale](#ADONETPersistenceRationale).
 
+__NB for Microsoft SQL Server users:__
+For the ADO.NET Storage Provider to work with SQL Server, you need to add [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient/) NuGet package to your project.
```

Comment:
Orleans will work with any ADO.NET provider as long as the back-end has been set up with the required tables and the required meta-data has been added in the code. Out of the box we provide setup scripts for SQL Server, MySQL and PostgreSQL. These are found [here](https://github.com/dotnet/orleans/tree/master/src/OrleansSQLUtils) and are the only ones that should be considered officially supported. Correcting my previous comment about supporting only SQL Server and MySQL. Nevertheless, I agree with @jdom about tests, I was under the impression that PostgreSQL tests were added. I wonder if @xclayl can say it's woroking.